### PR TITLE
Added Hound CI config File

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes: #174

Changes:
- Added config file for integartion of HoundCI

Benefits:
Hound reviews your code when a pull request is opened or updated. If Hound finds any style violations in the modified code it will comment on the appropriate line. It will be easier for the beginners to fix the issue and makes the life easier for reviewer and if we add flake8 in travis build than PR will fail due to violations in the PR so beginners will find difficult to debug and fix the issue

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@AnshulMalik Review again 😄

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+python:
+  enabled: true
+  config_file: .flake8.ini 
+
```
Comment (AnshulMalik) [inline .hound.yml]: Please add a newline

Comment (vaibhavsingh97) [reply .hound.yml]: i already added

Comment (vaibhavsingh97) [reply .hound.yml]: image

**Comment by cclauss** [comment]
LGTM  Travis was failing when this PR was submitted but it had nothing to do with the changes in this PR.

File: .hound.yml (hunk 1 above)
Comment (cclauss) [inline .hound.yml]: LGTM

File: .flake8.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 120
```
Comment (cclauss) [inline .flake8.ini]: .travis.yml now says 85 instead of 120 https://github.com/fossasia/query-server/blob/master/.travis.yml#L12

Comment (vaibhavsingh97) [reply .flake8.ini]: @cclauss I will fix this 👍

**Pull request merged**